mappings/net/minecraft/client/util/telemetry/GameLoadTimeEvent.mapping

```diff
@@ -0,0 +1,35 @@
+CLASS net/minecraft/class_8561 net/minecraft/client/util/telemetry/GameLoadTimeEvent
```

Comment:
Covered in #3559 as `TimeRecorder`. I also don't think `Event` suffix is proper, as this doesn't extend any jfr event or such.